rules/coding-style/tests/Rector/String_/UseClassKeywordForClassNameResolutionRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\String_\UseClassKeywordForClassNameResolutionRector\Fixture;
+
+class SomeClass
+{
+    public static function createCache()
+    {
+        return 'cache';
+    }
+}
+
+class AnotherClass
+{
+    private function write($first)
+    {
+
+    }
+
+    public function get()
+    {
+        $this->write(
+            '<?php if (Rector\CodingStyle\Tests\Rector\String_\UseClassKeywordForClassNameResolutionRector\Fixture\SomeClass::createCache($netteCacheStorage, %var,  $this->global->cacheStack, %node.array?)) { ?>'
+        );
+    }
+}
```

Comment:
Test fixture should be as simple as possible too. It save so much reading while fixing bugs months later.

Here local method could be called, e.g. like this:

```php
class SomeClass
{
    public function run()
	{
		return 'SomeClass::staticCall()';
	}

    public static function staticCall()
	{
	}
}
```

Reply:
Simplified as much as possible. If I don't use the fully qualified class name `class_exists` returns false